### PR TITLE
Ease of release: Replace `gulp` with `wp-svn` orb

### PR DESCRIPTION
**Pull request by kienstra**
## Why yet another GCB PR?
* We've been deploying with Gulp, which is outdated. And I made it too complex.
* Now, we're going to use the [wp-svn Orb](https://circleci.com/developer/orbs/orb/studiopress/wp-svn) that we're maintaining.

## Changes
* Remove Gulp, and replace it with the [wp-svn Orb](https://circleci.com/developer/orbs/orb/studiopress/wp-svn)
* Simplify `.circleci/config.yml`

## Testing instructions
Not needed

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -144,25 +136,12 @@ workflows:
...
       - wp-svn/check-versions:
-          changelog-file: CHANGELOG.md
-          filters:
-            tags:
-              only: /^\d+\.\d+\.\d+$/
-            branches:
-              ignore: /.*/
-      - svn-deploy:
-          context: genesis-svn
-          requires:
```
Comment (kienstra) [inline .circleci/config.yml:155]: Add this back in before merging

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,63 +0,0 @@
-# Contributor Covenant Code of Conduct
-
-## Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as
```
Comment (kienstra) [inline CODE_OF_CONDUCT.md:5]: This isn't a community project, so it's hard to see why we need a code of conduct.

Comment (kienstra) [reply CODE_OF_CONDUCT.md:5]: Instead of adding this to `.distignore`, I thought it'd be easier to delete it.

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
Thanks so much, @dreamwhisper!

**Pull request merged**